### ISSUE TITLE
Backpacks Should Not Fit in Backpacks

### ISSUE THREAD

**Issue by StrawberryMoses** [unitystation/unitystation#2339]
# Description

Currently, backpacks can be put into other backpacks. They should be too large to fit inside each other.

**Comment by chairbender** [comment]
it used to be like that, I think someone may have messed with backpack storage capacity or backpack item size

**Comment by Bod9001** [comment]
remember that the blue space backpack can store backpacks

**Comment by chairbender** [comment]
Good first issue. You'll just need to change ItemAttributes stuff on Backpack.prefab and/or BackpackCapacity to ensure that backpacks can't fit in each other.

Reference inventory system docs: https://unitystation.github.io/unitystation/development/Inventory-System/

**Comment by Xist3nce** [comment]
I believe either someone already fixed this or the base prefab for backpacks was what was spawned to check. I checked the traits and both the Gray Backpack and Captain's backpack prefab have their size trait to huge, and can't fit inside of eachother. However the base backpack prefab and backwear prefab were both set to tiny, which may have been what was spawned for this test. Regardless I've changed the value on the test PR I'll be putting up later whenever I get my bearings.
